devices/updateStatus.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const dynamodb = require('../lib/dynamodb');
+const utils = require('../lib/utils');
+var _ = require('underscore');
+const constants = require('../lib/constants');
+
+module.exports.updateStatus = (event, context, callback) => {
+
+  const input = JSON.parse(event.body);
+  const params = {
+    TableName: process.env.DEVICES_TABLE_NAME,
+    IndexName: 'DevicesChipIdIndex',
+    KeyConditionExpression: 'chipId = :chipId',
+    ExpressionAttributeValues: { ':chipId': input.chipId }
+  };
+
+  dynamodb.query(params, function (err, result) {
+    if (err) {
+      console.log(err);
+      callback(null, utils.createResponse(500, 'An internal server error occurred'));
+      return;
+    }
+    var attributeUpdates = utils.toAttributeUpdates({
+      status: constants.STATUS_OFFLINE,
+      updatedAt: new Date().getTime()
+    });
+    var result = JSON.stringify(result.Items);
```

Comment:
I don't think we need stringify here, result.Items should be an json array already. Also need to check if the result.length = = 1 or not. It should be 1 or raise error.

When length is 1, you can use result.Items[0], no need slicing